docs/integrations/sources/spacex-api.md

```diff
@@ -62,11 +62,11 @@ The SpaceX-API source connector supports the following [sync modes](https://docs
 
 ## API method example
 
-GET https://api.spacexdata.com/v5/launches/latest
+`GET https://api.spacexdata.com/v5/launches/latest`
 
 ## Performance considerations
 
-SpaceX's [API reference](https://api.spacexdata.com/v5/) has both v4 an v5 for launches. The connector as default uses V4 as it has minimal bugs.
+The SpaceX API has both v4 and v5 for [launches](https://github.com/r-spacex/SpaceX-API/tree/master/docs/launches). The connector as default uses V4 as it has minimal bugs.
```

Comment:
Hey @marcosmarxm, I saw that you reviewed the [PR](https://github.com/airbytehq/airbyte/pull/18311) for the SpaceX API and I was hoping you could help me with this URL. The URL in the documentation that was originally provided, https://api.spacexdata.com/v5/, seems to be dead and it looks like the website may have been taken down. I was wondering if you think it would be okay to use https://github.com/r-spacex/SpaceX-API/tree/master/docs/launches instead, or if you have any other suggestions for a better URL to point users to.

Reply:
I think the Github link is good. The previous one was only a redirect to Github one though